### PR TITLE
fix: only show priority  assignment in `zeebe:userTask`

### PR DESCRIPTION
**Pull request by Skaiir**
Related to [#21772](https://github.com/camunda/camunda/issues/21772)

### CODE REVIEW

**Review by nikku — COMMENTED** [review]
One minor nit-pick: I'd consolidate the "is this a relevant task" into a single utility.

File: test/spec/provider/zeebe/PriorityDefinitionProps.spec.js [hunk 1]
```diff
@@ -153,14 +171,14 @@ describe('provider/zeebe - PriorityDefinitionProps', function() {
...
-    it('should create non-existing extension elements and priority definition',
+    it('should create priority definition',
       inject(async function(elementRegistry, selection) {
 
         // given
         const userTask = elementRegistry.get('UserTask_2');
 
         // assume
-        expect(getBusinessObject(userTask).get('extensionElements')).to.not.exist;
+        expect(getExtensionElementsList(getBusinessObject(userTask), 'zeebe:priorityDefinition')[0]).to.not.exist;
```
Comment (nikku) [inline test/spec/provider/zeebe/PriorityDefinitionProps.spec.js:181]: Is this what we want to express?

```suggestion
        expect(getExtensionElementsList(getBusinessObject(userTask), 'zeebe:priorityDefinition')).to.be.empty;
```

Comment (Skaiir) [reply test/spec/provider/zeebe/PriorityDefinitionProps.spec.js:181]: Yeah this makes more sense.

**Review by nikku — APPROVED** [review]
Thanks for the fix :rocket:

**Comment by nikku** [comment]
We still need to ensure this field is removed when the user switches the types (via https://github.com/camunda/camunda-bpmn-js-behaviors).

**Comment by nikku** [comment]
Released via https://github.com/bpmn-io/bpmn-js-properties-panel/releases/tag/v5.23.2.

**Pull request merged**